geoips/create_plugin_registries.py

```diff
@@ -45,6 +46,38 @@
 LOG = logging.getLogger(__name__)
 
 
+def format_docstring(docstring, use_regex=True):
+    """Format the provided docstring placement in the plugin registry.
+
+    Found when using the CLI and inspecting the registry, some docstrings are formatted
+    in a hard to read manner and look pretty bad. This function will format these
+    docstrings to be easily readable, whether obtained via the CLI or manually inspected
+    in the plugin registry.
+
+    Parameters
+    ----------
+    docstring: str
+        - The docstring which we are going to format.
+    use_regex: bool, optional (default=False)
+        - Whether or not we want to apply regex formatting to the docstring. Usually
+          recommended as it will replace 'newline' chars but not purposeful
+          '.newline' strings.
+    """
+    if docstring is None:
+        return docstring
+    elif use_regex:
+        # Regex pattern for subbing out "\n" but not ".\n"
+        pattern = r"(?<!\.)\n"
+        docstring = re.sub(
+            pattern,
+            " ",
+            docstring.strip().replace("\n\n", "\n"),
+        )
+    else:
+        docstring = docstring.strip().replace("\n\n", "\n")
+    return docstring
```

Comment:
This is a nitpick, but this would be better written as:

```python
if docstring:
    if use_regex:
        # Regex pattern for subbing out "\n" but not ".\n"
        pattern = r"(?<!\.)\n"
        docstring = re.sub(
            pattern,
            " ",
            docstring.strip().replace("\n\n", "\n"),
        )
    else:
        docstring = docstring.strip().replace("\n\n", "\n")
return docstring
```

Reply:
Another question. Much of the time we add multiple new lines purposefully, to improve readability. Why remove them here?

Reply:
This is the format that I found to be most readable when looking at the output. Otherwise we are stuck with the newline problem in the docstring attribute that was poorly formatted. We can discuss exactly what we want at a future point. Happy to change this to what we all agree with.

Reply:
I also modified that function to what you posted above.